pkg/internalapis/cnsoperator/triggercsifullsync/v1alpha1/triggercsifullsync_types.go

```diff
@@ -0,0 +1,79 @@
+/*
+Copyright 2021 The Kubernetes authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// TriggerCsiFullSyncSpec is the spec for TriggerCsiFullSync
+type TriggerCsiFullSyncSpec struct {
+	// TriggerSync gives an option to trigger full sync on demand.
+	// Inorder to trigger a full sync, user can set any random value.
```

Comment:
As discussed in the meeting, using any random value to trigger a full sync is very confusing to the user.  Can this be improved?

Reply:
nit: Inorder -> In order